src/plugins/gres/gpu/gres_gpu.c

```diff
@@ -574,9 +578,16 @@ static void _add_fake_gpus_from_file(List gres_list_system,
 			      " that the format is <type>|<sys_cpu_count>|"
 			      "<cpu_range>|<links>|<device_file>", line_number);
 
+		bitstr_t *cpu_aff_mac_bitstr = bit_alloc(cpu_count);
```

Comment:
I thought slurm code style (linux kernel style) asks to declare everything at the beginning of the function.